internal/support/logging/types.go

```diff
@@ -47,7 +47,7 @@ func newPrivateLogger() privLogger {
 	return p
 }
 
-func (l privLogger) log(logLevel string, msg string, args ...interface{}) {
+func (l privLogger) log(logLevel string, msg string) {
```

Comment:
The changes in this file are too aggressive. Since PR #787 was merged, the args parameter should be used to specify key/value pairs for inclusion in our log messages. It no longer corresponds to labels and should be preserved. I don't think any changes in this file are required.